nautobot/extras/utils.py

```diff
@@ -267,6 +267,9 @@ def populate_model_features_registry():
         feature_name = lookup_conf["feature_name"]
         registry["model_features"][feature_name] = registry_items
 
+    if registry["model_features"].get("populate_model_features_registry_called", False):
+        registry["populate_model_features_registry_called"] = True
```

Comment:
Should this be?

```suggestion
        registry["model_features"]"populate_model_features_registry_called"] = True
```

?

Alternately, should we make this a top-level registry key instead of storing it under the `model_features` key?

Reply:
I didn't want to clash with this: https://github.com/nautobot/nautobot/blob/next/nautobot/extras/utils.py#L213-L214

Nor is `populate_model_features_registry_called` a model feature. I just wanted someplace to store globally that "you, the runtime engine, have been here before".

Reply:
My point was that you're **looking for** `"populate_model_features_registry_called"` inside of `registry["model_features"]`, but then **setting** `"populate_model_features_registry_called"` at the top level of `registry` - one way or the other, it seems wrong?

Reply:
Doh!

Reply:
Fixed. I meant for `registry.get("populate_model_features_registry_called", False)`